docs/src/pages/components/data-grid/scrolling/ScrollPlayground.tsx

```diff
@@ -0,0 +1,98 @@
+import * as React from 'react';
+import { Grid, Box, Button } from '@mui/material';
```

Comment:
Can we import each component from its own endpoint to give good example to the users ?

```suggestion
import { Box } from '@mui/material/Box';
import { Button } from '@mui/material/Button';
import { Grid } from '@mui/material/Grid';
```